trio/_file_io/_helpers.py

```diff
@@ -0,0 +1,19 @@
+import trio
+
+from functools import wraps, partial
```

Comment:
I have mixed feelings about `flake8` because it tends to produce lots of annoying false positives... pyflakes or pylint might be ok, but I suspect that [they'd take some work to configure](https://codewithoutrules.com/2016/10/19/pylint/) to work well, since trio does a bit more metaprogramming than static analysis tools tend to like.

I guess a simple incremental approach would be to start running one of them exclusively in find-unused-imports mode, and then maybe grow from there :-). I tried running them on the code and they do indeed spew tons of false positives, but also caught a whole bunch of unused imports...